Assets/Helika/Source/EventManager.cs

```diff
@@ -65,39 +62,46 @@ public async Task Init(string apiKey, string gameId, HelikaEnvironment env, bool
 
             _isInitialized = true;
 
-            // If Localhost is set, force disable sending events
-            _enabled = env != HelikaEnvironment.Localhost ? enabled : false;
+            // If Localhost is set, force print events
+            _printEventsOnly = printEventsOnly;
+            _telemetry = env != HelikaEnvironment.Localhost ? telemetryLevel : TelemetryLevel.None;
```

Comment:
We're saying if it's LocaslHost dont "send" any events? 
Wouldn't we still want the ability to log any telemetry level locally for testing? If I'm understanding correctly this will set telemetry to None if it's localhost

Reply:
okay, I'll update this.

Reply:
HelikaEnvironement indicates where data is being sent (Helika API prod or Helika API Develop). Localhost means we shouldn't send anything anywhere.